package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
```

Comment:
**[OPTIONAL]**
- [ ] You are using Github Flow for submitting this project but you were supposed to use Gitflow to submit this project means you always use the Github Flow approach when you have a Group Capstone Project because in that case there are multiple branches because more than one person works on it. But when it comes to the Solo capstone project you are supposed to create only one feature branch from your development branch and after the approval, you merge your feature branch into the development branch 😊